### PR TITLE
[receiver/dockerstats] Fix integration tests

### PR DESCRIPTION
**Pull request by mx-psi**
**Description:** 

Fixes dockerstats receiver integration tests. Split off from #17044

**Link to tracking Issue:** Relates to #17037

### CODE REVIEW

**Review by TylerHelmuth — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

File: receiver/dockerstatsreceiver/integration_test.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer/consumertest"
-	"go.opentelemetry.io/collector/receiver"
+	rcvr "go.opentelemetry.io/collector/receiver"
```
Comment (dmitryax) [inline receiver/dockerstatsreceiver/integration_test.go:34]: Why is this change needed? I don't see any `receiver` identifiers which it can conflict with

Comment (bogdandrutu) [reply receiver/dockerstatsreceiver/integration_test.go:34]: There is a type `receiver` in the receiver.go.
